modules/appeals_api/app/controllers/appeals_api/v2/decision_reviews/supplemental_claims_controller.rb

```diff
@@ -66,6 +65,12 @@ def schema
       ).schema(self.class::FORM_NUMBER)
     )
 
+    unless Flipper.enabled?(:decision_review_sc_pact_act_boolean)
+      response.tap do |s|
+        s.dig(*%w[definitions scCreate properties data properties attributes properties])&.delete('potentialPactAct')
+      end
+    end
+
```

Comment:
wondering if we should be "editing"the form data they send us...i guess it really does not matter, we delete the whole thing anyways eventually

Reply:
Oh, this is the `schema` endpoint, so it's a simple `GET`.

Reply:
Oh, i see now